examples/magento-graphcms/pages/cart.tsx

```diff
@@ -79,29 +81,21 @@ function CartPage() {
         <Container maxWidth='md'>
           <>
             {hasItems ? (
-              <Box sx={(theme) => ({ mt: theme.spacings.lg })}>
+              <>
                 <CartItems
-                  items={data?.cart?.items}
-                  id={data?.cart?.id ?? ''}
-                  key='cart'
-                  renderer={{
-                    BundleCartItem: CartItem,
-                    ConfigurableCartItem,
-                    DownloadableCartItem: CartItem,
-                    SimpleCartItem: CartItem,
-                    VirtualCartItem: CartItem,
-                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-                    // @ts-ignore GiftCardProduct is only available in Commerce
-                    GiftCardCartItem: CartItem,
+                  cart={data.cart}
+                  itemProps={{
+                    size: isMobile ? 'small' : 'large',
```

Comment:
useMediaQuery is not allowed in the examples directory.

I think it would be a good idea to create two props on CartItems directly: `sizeSm` / `sizeMd` and use the useMediaQuery inside the component.